src/NotExistsClassException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+namespace JakubOnderka\PhpParallelLint;
+
+class NotExistsClassException extends Exception
```

Comment:
Really odd name, Perhaps ClassNotFoundException, ClassNotFoundInFile?

Reply:
This feedback belongs with PR #44.